publify_core/app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -22,10 +22,10 @@
   padding: 45px;
   border: 1px solid #ccc;
 }
-.sidebar_item {
+.sidebar-item {
```

Comment:
Why is this necessary?

Reply:
It has be matched with other erb file we defined

Reply:
This does not have anything to do with the sidebar style bug in issue #1.

Reply:
Also, `sidebar-item` is not a class used anywhere.